src/server/replica.h

```diff
@@ -197,10 +230,11 @@ class Replica {
 
   std::shared_ptr<MultiShardExecution> multi_shard_exe_;
 
-  std::queue<std::pair<TranactionData, bool>> trans_data_queue_;
-  static constexpr size_t kYieldAfterItemsInQueue = 50;
-  ::util::fibers_ext::EventCount waker_;
   bool use_multi_shard_exe_sync_;
+  static constexpr size_t kYieldAfterItemsInQueue = 50;
+  // Pass entries from StableSyncDflyReadFb to StableSyncDflyExecFb.
+  TransactionDataQueue txd_queue_{kYieldAfterItemsInQueue};
+
```

Comment:
I've refactored the replica a little - the simpler the code is on a higher level, the easier it is to understand the flow